### PR TITLE
Improve the text of the Columns with docs

### PR DESCRIPTION
**Pull request by budnix**
### Context

Improve the text of the Columns with docs. Fix typos and unify the demos (use the same variable name for each example).

### Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. #7624 
2.
3.

### Affected project(s):
- [ ] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [x] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Comment by budnix** [comment]
🔔  @krzysztofspilka I need your review :)

File: docs/next/guides/columns/column-width.md [hunk 1]
```diff
@@ -68,7 +68,7 @@ const hot2 = new Handsontable(container, {
 
 ## Setting the column width using a function
 
-The column width can be set using a function. In this example, the size of all columns is set using a function by taking a column `index` (1, 2 ...) and multiplying it by `40px` for each consecutive column. 
+The column width can be set using a function. In this example, the size of all columns is set using a function by taking a column `index` (1, 2 ...) and multiplying it by `40px` for each consecutive column.
```
Comment (krzysztofspilka) [inline docs/next/guides/columns/column-width.md:71]: The column width can be set using a function.  > please remove.

**Review by krzysztofspilka — COMMENTED** [review]
Cool, nothing important really to be fixed.

File: docs/next/guides/columns/column-width.md [hunk 2]
```diff
@@ -20,7 +20,7 @@ tags:
 
 ## Overview
 
-By default, the column width adjusts to the width of the content. The minimum width is `50px`, including `1px` for borders on the sides. The column size can be passed as a `constant`, an `array`, or a `function`.
+By default, the column width adjusts to the width of the content. However, if the width of the column content is less than `50px`, including `1px` for borders on the sides, the column width will remain constant at `50px`. The column size can be passed as a `constant`, an `array` or a `function`.
```
[the remark this answers is not in the document]
Comment (budnix) [reply docs/next/guides/columns/column-width.md]: Done in 4750b948fce55293009c9adc0a2ee379ad7d69e9

File: docs/next/guides/columns/column-width.md (hunk 1 above)
Comment (budnix) [reply docs/next/guides/columns/column-width.md]: Done in 4750b948fce55293009c9adc0a2ee379ad7d69e9

**Review by Siemienik — APPROVED** [review]
Code examples look great 👍 
@krzysztofspilka could you please review the texts?

File: docs/next/guides/columns/column-width.md [hunk 3]
```diff
@@ -20,7 +20,7 @@ tags:
 
 ## Overview
 
-By default, the column width adjusts to the width of the content. The minimum width is `50px`, including `1px` for borders on the sides. The column size can be passed as a `constant`, an `array`, or a `function`.
+By default, the column width adjusts to the width of the content. However, if the width of the column content is less than `50px`, including `1px` for borders on the sides, the column width remains constant at `50px`. The column size can be passed as a `constant`, an `array` or a `function`.
 
 The content inside a cell will be wrapped if it doesn't fit the cell's width.
```
Comment (krzysztofspilka) [inline docs/next/guides/columns/column-width.md:25]: This "will" bothers me, but let's leave it as it is for now.

**Review by krzysztofspilka — APPROVED** [review]
Looks good to me

**Pull request merged**